examples/Training/python/Create_Image.py

```diff
@@ -67,7 +71,7 @@ def planeGen():
 conn.getUpdateService().saveObject(p)
 
 
-# Create an Image from an existing image
+# Create an Image from an existing image:
 # =================================================================
 # We are going to create a new image by passing the method a 'generator' of 2D
 # planes This will come from an existing image, by taking the average of 2
```

Comment:
Also, blank line 90 below means that section is printed split into 2 code blocks at that point, which looks like a funny place to split.